acquia/mc-cs-plugin-custom-objects
# [MAUT-7878] Export Custom Item

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N]
| New feature/enhancement? (use the a.x branch)      | [ Y]
| Deprecations?                          | [ N]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ Y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | https://backlog.acquia.com/browse/MAUT-7878

#### Description:

This PR is to export customItems with linkedContactIds.



#### Steps to test this PR:
1. create a custom object with any number of fields.
2. create any number of custom items for newly created custom object in step 1. 
3. can link any number of custom items to contacts.(can link more than 1 contacts to any customItem).
4. beside new button in custom item list page , will view a button for export as csv, click on that.
5. Can check the email will receive a link to a file in zip format, click on the file, it will download the file. can view the contents and check with data we created in steps 1-3.

note:
1. linkedContactIds will be comma separated, if the custom item is linked to multiple contact ids or will be empty if no contact id is linked. please check https://github.com/acquia/mc-cs-plugin-custom-objects/wiki/Import for format of csv file.
